ios/RCTWebRTC/WebRTCModule+RTCPeerConnection.m

```diff
@@ -402,33 +402,56 @@ - (NSString *)statsToJSON:(RTCStatisticsReport *)report
         [s appendString:key];
         [s appendString:@"\":"];
         NSObject *statisticsValue = [statistics.values objectForKey:key];
-        if ([statisticsValue isKindOfClass:[NSArray class]]) {
-            [s appendString:@"["];
-            BOOL firstValue = YES;
-            for (NSObject *value in (NSArray *)statisticsValue) {
-              if(firstValue) {
+        [s appendString:[self appendValue:statisticsValue]];
+    }
+    
+    [s appendString:@"}]"];
+  } 
+
+  [s appendString:@"]"];
+
+  return s;
+}
+
+- (NSString *)appendValue:(NSObject*) statisticsValue {
+    NSMutableString *s = [NSMutableString stringWithCapacity:16 * 1024];
```

Comment:
Why create a new one? Let's pass around the original one when calling this function instead.